package.json

```diff
@@ -45,67 +50,77 @@
     "release:major": "node scripts/release/release.js major",
     "release:minor": "node scripts/release/release.js minor",
     "release:patch": "node scripts/release/release.js patch",
-    "test": "npm run jest && npm run compile && npm run nightwatch && npm run nightwatch:port-provided && wdio wdio.conf.js",
-    "wdio": "wdio wdio.conf.js"
+    "test": "npm run jest && npm run nightwatch && npm run nightwatch:port-provided && npm run wdio",
+    "wdio": "npm run wdio-webpack-obj && npm run wdio-webpack-func && npm run tt-wdio-webpack-obj && npm run tt-wdio-webpack-func",
+    "wdio-webpack-obj": "wdio",
```

Comment:
What's the idea behind this script name / script usage?

Reply:
These scripts are testing webpack configuration passed as a object which is toggle in wdio.config here